lib/rules/class-methods-use-this.js

```diff
@@ -42,6 +56,16 @@ module.exports = {
         }
 
         /**
+         * Check if the node is an instance method not excluded by config
+         * @param {ASTNode} node - node to check
+         * @returns {boolean} True if it is an instance method, and included
```

Comment:
It's not very important for sure, but when I saw the antonym I had to look again to make sure it was actually the same, and that's time I could have avoided spending. :stuck_out_tongue: